include/enum_quda.h

```diff
@@ -182,8 +182,9 @@ typedef enum QudaMultigridCycleType_s {
 } QudaMultigridCycleType;
 
 typedef enum QudaSchwarzType_s {
-  QUDA_ADDITIVE_SCHWARZ,
-  QUDA_MULTIPLICATIVE_SCHWARZ,
+  QUDA_ADDITIVE_SCHWARZ = 0,
+  QUDA_ADDITIVE_MADWF_SCHWARZ = 1,
```

Comment:
I don't like taking advantage of `QudaSchwarzType` to distinguish between additive schwarz and MADWF schwarz; we could use a separate enum in a separate field that's MDWF-specific.

Reply:
https://github.com/lattice/quda/pull/1061/commits/5d9e47921fe9b8a5bcb89857ef843062baaba806